self_unenrollment_tool/management/commands/generate_lti_key.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+from Crypto.PublicKey import RSA
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The pyCrypto library and its module RSA are no longer actively maintained and have been deprecated. Consider using pyca/cryptography library. https://bandit.readthedocs.io/en/latest/blacklists/blacklist_imports.html#b413-import-pycrypto

Reply:
This is a good catch! Will rewrite this bit to use `cryptography` instead of `pycrypto`/`pycryptodome`.